### ISSUE TITLE
Damage Multiplier getting applied twice, all damage that uses more+multipliers is wrong in pob

### ISSUE THREAD

**Issue by Senorcrunch** [PathOfBuildingCommunity/PathOfBuilding#7555]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### Check for support

- [X] I've checked that the calculation is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the value from the calculation in-game?

I can't be bothered working my way through all these questions, this is easy. In the calculations in PoB, it's showing the "more" total inclusive of the multiplier, then also multiplying again by the multiplier right after.

Look at the attached picture of PoB, you can reproduce this with any skill that has any added more and multiplier

I'm playing bane/blight so I tested those, but for this I've just shown Bane. I've made the pob match the images in terms of what i had enabled etc.

### What is the value from the calculation in Path of Building?

n/a

### How to reproduce the issue

n/a

### Character build code

```shell
https://pastebin.com/Rwp53hit
```

### Screenshots

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/79117428/18853e92-4708-4155-bf1d-2eea683b0c0e)
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/79117428/5558d628-2210-4ae2-a20b-597992033218)

**Comment by Wires77** [comment]
PoE tooltip doesn't take into account the Effective DPS Mod.  If you change the Calculation Mode to something other than "Effective DPS", the numbers will more closely match up.

Additionally, the more total is not inclusive of the multiplier, I'm not sure how you're determining that.  The 2.85 multiplier is your more multipliers multiplied together, not added.  I.e. 1.88 * 1.52 = 2.85.

> I can't be bothered working my way through all these questions, this is easy

In general, try not to claim a huge portion of the project is doing something wrong without applying at least the minimum effort to be sure you're right.  Especially when you call it easy.

**Comment by Senorcrunch** [comment]
Hi there,

Thanks for replying and confirming I'm an idiot. I ran and re-ran my calculations several times and now I cannot work out how I got it wrong. I also profusely apologise for you having to waste your time checking this.

> In general, try not to claim a huge portion of the project is doing something wrong without applying at least the minimum effort to be sure you're right. Especially when you call it easy.
Easy to check. My problem with the template approach to the bug layout is what I was trying to explain didn't fit well into the templated questions. Further to this, I wasn't entirely sure what bug type this would come under.
